k8s/oidc-aws/oidc-dp-configmap.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: oidc-discovery-provider
+  namespace: spire
+data:
+  oidc-discovery-provider.conf: |
+    log_level = "INFO"
+    # TODO: Replace MY_DISCOVERY_DOMAIN with the FQDN of the Discovery Provider that you will configure in DNS
+    domain = "MY_DISCOVERY_DOMAIN"
+    acme {
+        directory_url = "https://acme-staging-v02.api.letsencrypt.org/directory"
```

Comment:
I think this needs to be the production value `https://acme-v01.api.letsencrypt.org/directory`